x/crosschain/keeper/msg_server_add_to_outtx_tracker_test.go

```diff
@@ -283,7 +280,7 @@ func setupTssAndNonceToCctx(k *keeper.Keeper, ctx sdk.Context, chainId, nonce in
 	k.GetObserverKeeper().SetTSS(ctx, observerTypes.TSS{
 		TssPubkey: tssPubKey,
 	})
-	k.SetPendingNonces(ctx, types.PendingNonces{
+	k.GetObserverKeeper().SetPendingNonces(ctx, observertypes.PendingNonces{
```

Comment:
Why `SetPendingNonces` was still used in `crosschain`, did we forget to remove these functions?

Reply:
yes